### PR TITLE
Fixes for making FTP file source work in sdk and cluster

### PR DESCRIPTION
**Pull request by CuriousVini**
Issue: https://issues.cask.co/browse/CDAP-11629
Build: https://builds.cask.co/browse/CDAP-RUT1074-5

Tested SFTP on sdk and cluster.

### CODE REVIEW

File: cdap-master/src/main/java/org/apache/hadoop/fs/ftp/FTPInputStream.java [hunk 1]
```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
```
Comment (albertshau) [inline cdap-master/src/main/java/org/apache/hadoop/fs/ftp/FTPInputStream.java]: 2017

File: cdap-master/src/main/java/org/apache/hadoop/fs/sftp/SFTPInputStream.java [hunk 2]
```diff
@@ -0,0 +1,135 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
```
Comment (albertshau) [inline cdap-master/src/main/java/org/apache/hadoop/fs/sftp/SFTPInputStream.java]: is this supposed to have cask copyright?

File: cdap-master/src/main/java/org/apache/hadoop/fs/sftp/SFTPInputStream.java [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+import com.jcraft.jsch.JSchException;
+import com.jcraft.jsch.Session;
+import org.apache.hadoop.fs.FSInputStream;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.util.StringUtils;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+/** SFTP FileSystem input stream. */
```
Comment (albertshau) [inline cdap-master/src/main/java/org/apache/hadoop/fs/sftp/SFTPInputStream.java]: should this have a similar javadoc to FTPInputStream?

**Comment by gokulavasan** [comment]
@CuriousVini @albertshau Should copied files have our copyright or the original copyright?

**Review by albertshau — APPROVED** [review]
lgtm

**Comment by albertshau** [comment]
The other classes we have copied and modified have our copyright, so I think that's how we do it.

**Comment by CuriousVini** [comment]
Build is green. Merging.

**Pull request merged**